src/lib/rule2A_test.ts

```diff
@@ -0,0 +1,124 @@
+import {rule2A} from './rule2A';
+import {html, render} from 'lit-html';
+
+describe('The function for rule 2A', () => {
+
+  let container: HTMLElement;
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+  });
+
+  it('returns empty string for hidden elems that aren\'t referenced', () => {
+    render(html`<div id="foo" hidden>Hello world</div>`, container);
+    const elem = document.getElementById('foo');
+    expect(rule2A(elem!, {})).toBe('');
+  });
+
+  it('returns null for hidden elems that are referenced in an aria-labelledby', () => {
+    render(html`<div id="foo" hidden>Hello world</div>`, container);
+    const elem = document.getElementById('foo');
+    expect(rule2A(elem!, {ariaLabelledbyReference: true})).toBe(null);
+  });
+
+  it('returns null for hidden elems that are referenced by a label element', () => {
+    render(html`<div id="foo" hidden>Hello world</div>`, container);
+    const elem = document.getElementById('foo');
+    expect(rule2A(elem!, {labelReference: true})).toBe(null);
+  });
+
+  it('returns null for elem that is not hidden', () => {
+    render(html`<div id="foo">Hello world</div>`, container);
+    const elem = document.getElementById('foo');
+    expect(rule2A(elem!, {})).toBe(null);
+  });
+
+  it('considers aria-hidden', () => {
```

Comment:
starting from this test, all the remaining cases are testing `isHidden` directly.

you can use an export named TEST_ONLY to export the function for the test

```typescript
export const TEST_ONLY = {isHidden};
```

Reply:
I'm not sure what you mean here, would I test isHidden directly in the _test.ts file referring to it as TEST_ONLY?

When I try:
```
it('considers aria-hidden', () => {
    render(html`<div id="foo" aria-hidden="true">Hello world</div>`, container);
    const elem = document.getElementById('foo');
    expect(TEST_ONLY(elem!)).toBe(true);
  });
```
I get an error saying that TEST_ONLY is not callable.